.pipelines/azure-pipelines.yml

```diff
@@ -38,6 +38,8 @@ stages:
           Pool: 1es-dcv3
           Environment:
             ENABLE_PREFIX_TREE: OFF
+          FunctionalTestArguments: '--enable-perf'
```

Comment:
If this slows down CI too much we may also introduce a separate pipeline that runs on commits to `main`, not in PRs.